lib/hexpm_web/views/view_helpers.ex

```diff
@@ -348,6 +348,14 @@ defmodule HexpmWeb.ViewHelpers do
   def include_if_loaded(output, key, struct, view, name, assigns) do
     Map.put(output, key, Phoenix.View.render_one(struct, view, name, assigns))
   end
+
+  def auth_qr_code_svg(user) do
+    secret = Map.fetch!(user.tfa, :secret)
+
+    "otpauth://totp/hex.pm:#{user.username}?issuer=hex.pm&secret=#{secret}"
+    |> EQRCode.encode()
+    |> EQRCode.svg(width: 400)
```

Comment:
```suggestion
    |> EQRCode.svg(width: 250)
```

let's make this smaller so 1password code scanner gizmo can pick it up :joy:

<img width="503" alt="image" src="https://user-images.githubusercontent.com/76071/76607746-c9589b00-6514-11ea-86aa-6688ff2fe560.png">